packages/kit/src/views/Setting/components/ApiEndpointDialog/components/ApiEndpointList/ApiEndpointList.tsx

```
@@ -0,0 +1,227 @@
+          onSuccess={() => {
+            void refreshData();
+            void d.close();
+          }}
+          onCancel={() => {
+            void d.close();
+          }}
+        />
+      ),
+      showFooter: false,
+    });
+  }, [refreshData]);
```

ESLint

[error] React Hook useCallback has a missing dependency: 'intl'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)